tools/openshift-ci/Dockerfile.centos

```diff
@@ -11,12 +11,11 @@ RUN set -x && \
     SCL_BASE_PKGS="centos-release-scl scl-utils-build" && \
     INSTALL_PKGS="rh-ruby26 rh-ruby26-ruby-devel rh-git218 bsdtar" && \
     yum -y update && \
-    yum install -y --enablerepo=centosplus $SCL_BASE_PKGS && \
-    yum install -y --enablerepo=centosplus $INSTALL_PKGS && \
-    GECKODRIVER_DOWNLOAD_URL="$(curl -sSL https://api.github.com/repos/mozilla/geckodriver/releases/latest | grep -E 'browser_download_url.*linux64' | sed -E 's/.*(https[^"]*).*/\1/')" && \
+    yum install -y --enablerepo=centosplus --setopt=skip_missing_names_on_install=False $SCL_BASE_PKGS && \
+    yum install -y --enablerepo=centosplus --setopt=skip_missing_names_on_install=False $INSTALL_PKGS && \
+    GECKODRIVER_DOWNLOAD_URL="$(curl -sSL https://api.github.com/repos/mozilla/geckodriver/releases/latest | grep -Eo 'http.*linux64.tar.gz' | sed -E 's/.*(https[^"]*).*/\1/')" && \
```

Comment:
The root cause is that we get the output from curl (not sure it's a curl issue or server side issue) in one-line (unusual) sometimes, and in multi-lines the other times.

Reply:
Thanks! It should be server a side change.

btw it will save a little time if we had 
```
 yum install -y --enablerepo=centosplus --setopt=skip_missing_names_on_install=False $SCL_BASE_PKGS $INSTALL_PKGS
```

instaed of two lines. But anyway, we can change later.